GG/GG/Font.h

```diff
@@ -147,6 +147,9 @@ class GG_API Font
             pair.second. */
         Substring(const std::string& str_, const IterPair& pair);
 
+        /** Rebind to a different string.*/
```

Comment:
Maybe already done in a different PR that includes the same additions, but some documentation / explanation of this / these function(s) would be helpful.